app/AccountancyModule/UnitAccountModule/templates/Cashbook/default.latte

```diff
@@ -1,20 +1,19 @@
 {block #content}
-{include ../ActionMenu.latte}
-<h2 n:inner-block="title">{$unitPairs[$unitId]} - evidence plateb</h2>
+    <h1 n:inner-block="title">{$unitPairs[$unitId]}</h1>
+    {include ../header.latte}
 
-<div n:snippet="paragony">
-    {control cashbook}
-</div>
+    <div n:snippet="paragony">
+        {control cashbook}
+    </div>
     {control activateCashbookDialog}
     {control createCashbookDialog}
 {/block}
 
 {block actionMenuToolbar}
```

Comment:
Pletu se, nebo tohle se teď nikde nezobrazuje?

Reply:
ASi se to fakt už nezobrazuje. Tak já to smažu jo? nenašel jsem to ani v aktualní verzi zobrazené

Reply:
Jj :+1: